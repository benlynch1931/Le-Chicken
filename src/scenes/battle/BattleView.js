import React from 'react';
import { View } from 'react-native';
import { widthPercentageToDP as wp, heightPercentageToDP as hp } from 'react-native-responsive-screen';

const BattleView = () => {
  return (
    < View
      style={{
        width: wp("100%"),
        height: hp("61.58%"),
        backgroundColor: 'rgb(199, 252, 177)',
        zIndex: '1'
      }}
    />
  )

}
export default BattleView;
