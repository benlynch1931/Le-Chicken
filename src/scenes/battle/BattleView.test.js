import React from 'react';
import TestRenderer, { create, act } from 'react-test-renderer';
import BattleView from './BattleView.js';


describe('BattleView', () => {

  it('renders correctly', () => {
    const battle = create(<BattleView />)
    expect(battle).toMatchSnapshot();
  })

})
