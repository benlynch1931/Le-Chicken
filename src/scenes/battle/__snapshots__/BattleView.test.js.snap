// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BattleView renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgb(199, 252, 177)",
      "height": 821.5,
      "width": 750,
      "zIndex": "1",
    }
  }
/>
`;
